### PR TITLE
Fix confusion about how is translated "Reorder"

### LINKED ISSUE #28774: Confusing button text when you want to rearrange products in the category
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

This issue is for Pull Request from this URL:
https://github.com/PrestaShop/PrestaShop/pull/28343

In French, Polish, and some other languages, because of using "Reorder" as a original phrase, the button that allows you to sort the products from the category is not translated correctly.

### Expected behavior

In Polish, this button is translated as "Zamów ponownie" which means "Place your order again", when in the reality it should be "Zmień kolejność", "Change the order of the products", or... "Rearrange" as discussed with @Julievrz 

### Steps to reproduce

1. Catalog -> Products
2. Show products only from one category by filtering them.
3. Now you should be able to click the button allowing a merchant to sort the products

### PrestaShop version(s) where the bug happened

1.7

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by SLRkingRing**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update "Reorder" label for products_filter_position_asc  in listing product Backoffice to "Rearrange" for avoid confusion when translated. <br>In french for exemple, the traduction for Reorder is "Commander à nouveau" and it creates confusion in Backoffice to update product position.
| Type?             | bug fix 
| Category?         | BO
| BC breaks?        |  no
| Deprecations?     | no
| Fixed ticket?     | 
| Related PRs       | Fixes #28774
| How to test?      | Connect to BO, go to catalog > product , filter by one category, and the "Reorder" button name has changed now it's "Rearrange"
| Possible impacts? | Impact possible on the other language

### CODE REVIEW

**Comment by mparvazi** [comment]
@Julievrz @SLRkingRing 
My suggestion: `Rearrange`

**Comment by kpodemski** [comment]
Nice catch. It's the same problem for Polish. Reorder is translated to the text saying "Place an order again" :)

**Review by matthieu-rolland — COMMENTED** [review]
It's confusing indeed, I'd go for `rearrange` too, but @Julievrz is the one to decide ;)

**Review by Progi1984 — DISMISSED** [review]

**Comment by Julievrz** [comment]
Hi, I think "rearrange" is more relevant here. Thank you for raising this wording issue!

File: src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig]:
```suggestion
                <input type="button" class="btn btn-outline-secondary" name="products_filter_position_asc" value="{{ "Rearrange"|trans({}, 'Admin.Actions') }}" onclick="productOrderPrioritiesTable();" />
```

Comment (Julievrz) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig]: After verification, I already see "Rearrange" in English, 1.7.8.5 🤔

Comment (matks) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig]: @Julievrz So do we still go with "Rearrange" ? Or another word needs to be chosen?

**Review by matks — CHANGES_REQUESTED** [review]
Waiting for @Julievrz answer 👍 but thank you @SLRkingRing for contribution

File: src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig
Comment (Julievrz) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig]: "Rearrange" is ok, but I don't understand because when I check the BO in 1.7.8.5 I can already see the wording "Rearrange".

Comment (matks) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig]: @Julievrz I'm not sure what you mean

- if you open this back office page with 1.7.8.5 you see the wording being used is "Rearrange", not "Reorder" ?
- if you open the "Translation management" back office page with 1.7.8.5 you see the wording "Rearrange" is already registered which would mean it's already being used in the software?

Comment (Julievrz) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig]: The first option. When I went to the BO (1.7.8.5) the wording was already "Rearrange".

Comment (Julievrz) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig]: Update: the wording is "Rearrange" because I put "Rearrange" as a translation 5 months ago. So I confirm we can go with Rearrange. Sorry for the misunderstanding.

Comment (matks) [reply src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig]: OK 👍

**Review by matks — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Hlavtox** [comment]
Yeah the same thing in czech language. The button is called "Znovu objednat" = "Buy again" 😄

**Review by matks — APPROVED** [review]

**Comment by florine2623** [comment]
Hello @SLRkingRing ,

Thanks for the PR !

Could you create an issue please ? 
Product team need to approve it and it will be easier to test for the QA team.

Thanks !

**Comment by kpodemski** [comment]
@florine2623 I've just created an issue to speed things up

**Review by florine2623 — COMMENTED** [review]
Hello @SLRkingRing ,

Thanks for the PR !

I have the right word now :

Screenshot 2022-06-20 at 10 13 05

But the translation is missing :
Screenshot 2022-06-20 at 10 58 41

Could you check that please ? :)
Thanks!

**Comment by kpodemski** [comment]
@PrestaShop/prestashop-maintainers 

I think translation won't be available before regenerating translations catalog?

**Review by florine2623 — APPROVED** [review]
ok @kpodemski ^^
Then, it is QA ✅ !
Thanks all !

**Comment by Progi1984** [comment]
Thanks @SLRkingRing & @florine2623

**Pull request merged**
